src/platform/imx8m/include/platform/drivers/interrupt.h

```diff
@@ -61,8 +61,7 @@
 /* 32 HW interrupts + 8 IRQ_STEER lines each with 64 interrupts */
 #define PLATFORM_IRQ_HW_NUM	XCHAL_NUM_INTERRUPTS
 #define PLATFORM_IRQ_CHILDREN	64 /* Each cascaded struct covers 64 IRQs */
-/* IMX: Covered steer IRQs are modulo-64 aligned. */
-#define PLATFORM_IRQ_FIRST_CHILD  0
+#define PLATFORM_IRQ_FIRST_CHILD  PLATFORM_IRQ_HW_NUM
 
```

Comment:
Is this to be placed in a separate patch?

Reply:
I think I should add a patch specifically dedicated to enabling the 8MP. But that would make an intermediate version which doesn't compile.

Reply:
We could keep this as it is in the patches that does the separation. Meaning that the code for 8MP will compile but will not work. And then next patch.

Or you could leave it in this patch but add a comment to explain why IRQ_FIRST_CHILD is no longer 0.

Reply:
I feel like there may be a better way to place the comment, for example why on the other platform it is different (AFAIK only i.MX8/i.MX8X has it different from PLATFORM_IRQ_HW_NUM).